<h2>Introduction</h2>
<html><body><p id="p0030">The tumor suppressor BRCA1 coordinates two key steps during DNA double-strand break (DSB) repair by homologous recombination (HR) (<a class="workspace-trigger" href="#bib13">Chen et al., 2018</a>). Initially, BRCA1 promotes 5′-3′ end resection, and subsequently, it recruits the PALB2/BRCA2 mediator complex to load the RAD51 recombinase onto 3′ single-stranded DNA (ssDNA). In BRCA1-deficient cells, HR is severely compromised, leading to genome instability during development that precipitates tumorigenesis (<a class="workspace-trigger" href="#bib13">Chen et al., 2018</a>). However, 53BP1 loss largely restores HR in BRCA1-deficient mice, resulting in the rescue of the embryonic lethality and the suppression of cancer predisposition in pre-clinical mouse tumor models (<a class="workspace-trigger" href="#bib5">Bouwman et al., 2010</a>, <a class="workspace-trigger" href="#bib7">Bunting et al., 2010</a>, <a class="workspace-trigger" href="#bib12">Cao et al., 2009</a>). The dramatic rescue of HR in BRCA1-deficient mice is in part due to the role of 53BP1 in blocking end resection (<a class="workspace-trigger" href="#bib7">Bunting et al., 2010</a>). However, in some cell lines, BRCA1 depletion had minimal effects on end processing under conditions in which RAD51 filament formation was compromised (<a class="workspace-trigger" href="#bib55">Zhou et al., 2014</a>). RAD51 filament formation in BRCA1-deficient cells requires the alternative recruitment of PALB2/BRCA2 through RNF168-mediated chromatin ubiquitylation (<a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>). This raises the question of whether 53BP1, in addition to blocking resection, also antagonizes RNF168-driven RAD51 filament formation.</p><p id="p0035">The pro-end-joining and anti-recombination functions of 53BP1 require phospho-dependent interactions with downstream effectors such as RIF1 and PTIP (<a class="workspace-trigger" href="#bib45">Setiaputra and Durocher, 2019</a>). Recently, independent laboratories have discovered components of a “shieldin complex,” which acts downstream of RIF1 to block resection (<a class="workspace-trigger" href="#bib18">Dev et al., 2018</a>, <a class="workspace-trigger" href="#bib22">Findlay et al., 2018</a>, <a class="workspace-trigger" href="#bib23">Gao et al., 2018</a>, <a class="workspace-trigger" href="#bib24">Ghezraoui et al., 2018</a>, <a class="workspace-trigger" href="#bib26">Gupta et al., 2018</a>, <a class="workspace-trigger" href="#bib35">Mirman et al., 2018</a>, <a class="workspace-trigger" href="#bib38">Noordermeer et al., 2018</a>). Loss of RIF1 or shieldin leads to defective immunoglobulin class switch recombination (CSR), blocks the fusion of dysfunctional telomeres, and promotes poly ADP ribose polymerase inhibitor (PARPi) resistance in BRCA1-deficient cells (<a class="workspace-trigger" href="#bib18">Dev et al., 2018</a>, <a class="workspace-trigger" href="#bib22">Findlay et al., 2018</a>, <a class="workspace-trigger" href="#bib23">Gao et al., 2018</a>, <a class="workspace-trigger" href="#bib24">Ghezraoui et al., 2018</a>, <a class="workspace-trigger" href="#bib26">Gupta et al., 2018</a>, <a class="workspace-trigger" href="#bib35">Mirman et al., 2018</a>, <a class="workspace-trigger" href="#bib38">Noordermeer et al., 2018</a>, <a class="workspace-trigger" href="#bib45">Setiaputra and Durocher, 2019</a>). However, RIF1 and PTIP also have essential functions in replication and transcription (<a class="workspace-trigger" href="#bib1">Alver et al., 2017</a>, <a class="workspace-trigger" href="#bib8">Buonomo et al., 2009</a>, <a class="workspace-trigger" href="#bib14">Cho et al., 2003</a>, <a class="workspace-trigger" href="#bib17">Daniel et al., 2010</a>), making their roles in 53BP1-dependent DSB repair ambiguous.</p><p id="p0040">To determine the contribution of PTIP binding to 53BP1-dependent suppression of end resection and HR, we generated 53BP1-knockin mice harboring a single point mutation (S25A) in 53BP1. The 53BP1S25A mutation selectively impairs PTIP while it enhances the association of RIF1 with DSBs. The mutant 53BP1 protein is permissive for end resection sufficient to rescue the lethality of <em>BRCA1</em><sup><em>Δ11</em></sup> mice, but it nevertheless blocks RAD51 loading. Our study thereby uncovers a new role for 53BP1 in antagonizing HR at the post-resection step.</p><p id="p0045">Finally, we report that distinct domains of BRCA1 are capable of counteracting anti-recombinogenic functions of 53BP1 pre- and post-resection. Thus, BRCA1 mutant cells lacking exon 11 are severely impaired in DSB resection, while the loss of the BRCA1 RING domain maintains resection capability but fails to support RAD51 foci formation. Since pathogenic germline mutations in the RING domain and exon 11 are frequently found in familial breast and ovarian cancers and may predict poor responses to chemotherapy (<a class="workspace-trigger" href="#bib20">Drost et al., 2016</a>, <a class="workspace-trigger" href="#bib52">Wang et al., 2016</a>), these results have implications for our understanding of cellular mechanisms leading to PARPi and platinum resistance.</p></body></html><h2>Results</h2>
<h3>53BP1S25A Mutation Increases the Association of RIF1 with DNA Breaks</h3>
<html><body><p id="p0050">To dissect the contribution of phospho-dependent 53BP1-PTIP interaction to DSB repair in a physiological setting that does not compromise replication, MLL3/4 methyltransferase activity (<a class="workspace-trigger" href="#bib25">Gong et al., 2009</a>, <a class="workspace-trigger" href="#bib42">Ray Chaudhuri et al., 2016</a>), or 53BP1-dependent p53 activation (<a class="workspace-trigger" href="#bib16">Cuella-Martin et al., 2016</a>), we replaced serine 25 with alanine in embryonic stem cells (<a class="workspace-trigger" href="#mmc1">Figure S1</a>A), which were used to generate <em>53BP1</em><sup><em>S25A</em></sup> mice. As expected, homozygous mutant <em>53BP1</em><sup><em>S25A</em></sup> mice were born at normal Mendelian frequencies and did not exhibit any overt phenotype (<a class="workspace-trigger" href="#mmc1">Figure S1</a>A; data not shown). 53BP1 protein was present at normal levels but irradiation-induced S25 phosphorylation, which is largely ATM dependent, was absent in <em>53BP1</em><sup><em>S25A</em></sup> mice (<a class="workspace-trigger" href="#mmc1">Figures S1</a>B and S1C). Notably, irradiation-induced foci (IRIF) for RIF1 were significantly elevated in <em>53BP1</em><sup><em>S25A</em></sup> MEFs (<a class="figure_ref" href="#fig1">Figure 1</a>A), while PTIP recruitment to DSBs was undetectable in mutant cells (<a class="figure_ref" href="#fig1">Figure 1</a>B). Consistent with these findings, analysis by mass spectrometry revealed that RIF1 peptides were enriched and PTIP peptides were depleted in 53BP1 immunoprecipitates from cells expressing 53BP1<sup>S25A</sup> compared to 53BP1<sup>WT</sup> (<a class="figure_ref" href="#fig1">Figures 1</a>C and 1D; <a class="workspace-trigger" href="#mmc2">Table S1</a>). Thus, the 53BP1S25A mutation specifically impairs PTIP while enhancing the association of RIF1 with DNA breaks. Conversely, it was shown that the mutation of seven N-terminal phosphorylation sites in 53BP1 that decreased RIF1-53BP1 association led to a concomitant increase in PTIP-53BP1 interaction (<a class="workspace-trigger" href="#bib9">Callen et al., 2013</a>).</p><figure id="fig1"></figure></body></html><h3>53BP1S25A Rescues the Lethality of BRCA1Δ11 Mice but Leads to Premature Aging</h3>
<html><body><p id="p0055">The complete loss of 53BP1 is thought to rescue the embryonic lethality of BRCA1-deficient mice by increasing DSB resection (<a class="workspace-trigger" href="#bib7">Bunting et al., 2010</a>) and by permitting RNF168-mediated loading of RAD51 onto ssDNA (<a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>). To examine the relative contribution of each of these processes to embryonic viability, we crossed <em>53BP1</em><sup><em>S25A</em></sup> mice with mice expressing a mutant form of BRCA1 where exon 11 is deleted. While homozygous <em>BRCA1</em><sup><em>Δ11</em></sup> mutation is incompatible with viability, we found that <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> mice were born at expected Mendelian frequencies and reached adulthood (<a class="figure_ref" href="#fig2">Figures 2</a>A and 2B). Similar to <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>−/−</em></sup> mice (<a class="workspace-trigger" href="#bib7">Bunting et al., 2010</a>), mutant males were sterile and showed reduced testes size (<a class="workspace-trigger" href="#mmc1">Figure S2</a>A), most likely reflecting the role of BRCA1 in meiotic sex chromosome inactivation (<a class="workspace-trigger" href="#bib50">Turner et al., 2005</a>). However, in striking contrast to <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>−/−</em></sup> mice, which exhibit a normal lifespan (<a class="workspace-trigger" href="#bib7">Bunting et al., 2010</a>), <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> mice died prematurely, with a median survival of 6 months and a maximum lifespan of 92 weeks (<a class="figure_ref" href="#fig2">Figure 2</a>B). <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> mice developed pathologies that are consistent with accelerated aging, including kyphosis, blindness, inflammation, hair graying, and overall reduced activity, with only a modest rate of cancer formation (&lt;10%) (<a class="figure_ref" href="#fig2">Figures 2</a>C and <a class="workspace-trigger" href="#mmc1">S2</a>B). Thus, a single-point mutation in 53BP1 disrupting PTIP binding is sufficient to fully rescue the prenatal lethality of <em>BRCA1</em><sup><em>Δ11</em></sup> mice, but it is insufficient to allow normal organismal aging.</p><figure id="fig2"></figure><p id="p0060">Premature aging could be due to elevated DNA damage signaling leading to senescence and apoptosis (<a class="workspace-trigger" href="#bib37">Niedernhofer et al., 2018</a>). Already at embryonic day 13.5 (E13.5), <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> embryos appear smaller than wild type (WT) (<a class="figure_ref" href="#fig2">Figure 2</a>D), and mutant primary murine embryonic fibroblasts (MEFs) prematurely senesced at the same rate as <em>BRCA1</em><sup><em>Δ11</em></sup> cells (<a class="figure_ref" href="#fig2">Figure 2</a>E). Various tissues from mutant embryos, including cerebellum, liver, and limb, showed TUNEL<sup>+</sup> and caspase 3<sup>+</sup> staining indicative of apoptosis (<a class="workspace-trigger" href="#mmc1">Figure S2</a>C); and increased apoptosis was associated with enhanced levels of DNA damage signaling, as measured by Kap1 phosphorylation (<a class="workspace-trigger" href="#mmc1">Figure S2</a>D). Consistent with the increased neural apoptosis, mutant mice had a smaller brain, although its overall structure was undisturbed (<a class="workspace-trigger" href="#mmc1">Figure S2</a>E). Thus, despite the loss of embryonic cells <em>in vivo</em> and poor growth <em>in vitro</em>, the rescue of viability in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> mice was associated with normal developmental processes but accelerated postnatal aging.</p></body></html><h3>Cells from BRCA1Δ1153BP1S25A Mice Are Severely Compromised for RAD51 Loading</h3>
<html><body><p id="p0065">Because RAD51-dependent HR is largely rescued in <em>BRCA1</em><sup><em>Δ11</em></sup> mice lacking 53BP1, mutant cells are resistant to PARPi and show robust RAD51 foci formation (<a class="workspace-trigger" href="#bib7">Bunting et al., 2010</a>). In striking contrast, <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> mice (n = 3) died within 9 days after treatment with PARPi. Histological analysis revealed severe injuries to the gastrointestinal tract, including the blunting of villi and the almost complete obliteration of stem cell crypts (<a class="figure_ref" href="#fig3">Figure 3</a>A). Consistent with the <em>in vivo</em> response, mutant B cells and MEFs showed hypersensitivity to PARPi, as indicated by the elevated induction of chromosomal aberrations and increased formation of radials (<a class="figure_ref" href="#fig3">Figures 3</a>B and <a class="workspace-trigger" href="#mmc1">S3</a>A). Mutant MEFs also showed a marked reduction in colony growth after PARPi treatment (<a class="figure_ref" href="#fig3">Figure 3</a>C). Moreover, we observed a severe impairment in ionizing radiation (IR)-induced RAD51 foci formation in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> MEFs and B cells (<a class="figure_ref" href="#fig3">Figures 3</a>D and <a class="workspace-trigger" href="#mmc1">S3</a>B). Notably, the HR defect is more severe in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells compared to that observed in BRCA1/PTIP-deficient cells (<a class="workspace-trigger" href="#bib9">Callen et al., 2013</a>), suggesting that PTIP has additional roles within the MLL3/4 complex independent of its interaction with 53BP1 (<a class="workspace-trigger" href="#bib42">Ray Chaudhuri et al., 2016</a>). Moreover, in contrast to 53BP1 ablation (<a class="workspace-trigger" href="#bib7">Bunting et al., 2010</a>), the 53BP1S25A mutation rescues the lethality of <em>BRCA1</em><sup><em>Δ11</em></sup> mice without restoring HR. Our results therefore indicate that severely reduced HR is compatible with viability but leads to premature aging.</p><figure id="fig3"></figure></body></html><h3>53BP1S25A Mutation Increases End Resection in BRCA1Δ11 Cells</h3>
<html><body><p id="p0070">We speculated that the mutant 53BP1S25A protein may allow end resection sufficient to rescue the lethality of <em>BRCA1</em><sup><em>Δ11</em></sup> mice, but inadequate for robust RAD51 foci formation or resistance to PARPi. To quantitate the level of end resection, we generated site-specific DSBs by inducible expression of the restriction enzyme AsiSI and performed END-seq, which maps the double-stranded DNA (dsDNA)/ssDNA junctions at DSB sites (<a class="workspace-trigger" href="#bib10">Canela et al., 2016</a>). We observed considerable variation in resection across individual sites targeted by AsiSI (<a class="figure_ref" href="#fig4">Figures 4</a>A and <a class="workspace-trigger" href="#mmc1">S4</a>A). Levels of end resection increased with breakage, which in turn correlated with active chromatin marks (<a class="workspace-trigger" href="#mmc1">Figures S4</a>B and S4C) (<a class="workspace-trigger" href="#bib3">Aymard et al., 2014</a>). By visual inspection, it was clear that MEFs derived from <em>53BP1</em><sup><em>−/−</em></sup> mice exhibited increased end resection relative to WT, whereas <em>BRCA1</em><sup><em>Δ11</em></sup> MEFs showed reduced nucleolytic processing at all DSB sites (<a class="figure_ref" href="#fig4">Figures 4</a>A and <a class="workspace-trigger" href="#mmc1">S4</a>A). The measurement of RPA-bound ssDNA by chromatin immunoprecipitation sequencing (ChIP-seq) (<a class="workspace-trigger" href="#bib6">Brick et al., 2018</a>), although less sensitive than END-seq, confirmed these findings (<a class="figure_ref" href="#fig4">Figures 4</a>A and <a class="workspace-trigger" href="#mmc1">S4</a>D).</p><figure id="fig4"></figure><p id="p0075"><em>53BP1</em><sup><em>S25A</em></sup> mutant MEFs showed slightly longer resection tracts than WT (<a class="figure_ref" href="#fig4">Figure 4</a>B and <a class="workspace-trigger" href="#mmc1">S4</a>E). Consistently, we observed mildly increased levels of chromatin-bound RPA in mutant cells (<a class="workspace-trigger" href="#mmc1">Figure S4</a>F). Moreover, single-strand annealing (SSA), a mutagenic RAD51-independent homology-mediated repair pathway that requires end resection (<a class="workspace-trigger" href="#bib46">Stark et al., 2004</a>), was slightly elevated (1.4-fold) in <em>53BP1</em><sup><em>S25A</em></sup> cells (<a class="workspace-trigger" href="#mmc1">Figure S4</a>G). The average resection length in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>−/−</em></sup> MEFs was similar to WT and intermediate between <em>53BP1</em> and <em>BRCA1</em> deficiency (<a class="figure_ref" href="#fig4">Figure 4</a>B). Despite severely compromised IR-induced RAD51 foci (<a class="figure_ref" href="#fig3">Figure 3</a>D), <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> MEFs accumulated elevated levels of ssDNA at AsiSI sites relative to <em>BRCA1</em><sup><em>Δ11</em></sup> or WT (<a class="figure_ref" href="#fig4">Figure 4</a>B). Moreover, IR-induced chromatin binding of RPA was increased in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells (<a class="workspace-trigger" href="#mmc1">Figure S4</a>F). Thus, disrupting the 53BP1-PTIP interaction largely rescues the end-resection defect associated with BRCA1 deficiency, but <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells are nevertheless defective in loading RAD51.</p></body></html><h3>End Resection Is Predominantly Catalyzed by DNA2 in BRCA1Δ1153BP1S25A Cells</h3>
<html><body><p id="p0080">The nucleases EXO1 and DNA2 are thought to act redundantly to promote end resection (<a class="workspace-trigger" href="#bib48">Symington, 2016</a>). Consistently, whereas depletion of EXO1 alone or treatment with the chemical inhibitor of DNA2 (NSC-105808) by itself did not decrease IR-induced RPA chromatin binding in replicating WT cells, end resection was significantly decreased when both nucleases were inactivated simultaneously (<a class="figure_ref" href="#fig4">Figure 4</a>C). However, in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells, treatment with DNA2 inhibitor (DNA2i) alone significantly reduced IR-induced RPA recruitment to damaged sites, while EXO1 inhibition had a smaller impact (<a class="figure_ref" href="#fig4">Figure 4</a>D). Similarly, BLM inhibition (BLMi) significantly reduced RPA foci formation in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells relative to WT (<a class="workspace-trigger" href="#mmc1">Figure S4</a>H). These data implied that DNA2 activity is normally blocked by the 53BP1-PTIP interaction. To determine which machinery catalyzes end resection in the absence of shieldin, we examined nucleolytic processing in <em>BRCA1</em><sup><em>Δ11</em></sup><em>SHLD3</em><sup><em>−/−</em></sup> cells and in <em>BRCA1</em><sup><em>Δ11</em></sup> cells depleted for RIF1. In contrast to <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells, DNA damage induced RPA chromatin binding in <em>BRCA1</em><sup><em>Δ11</em></sup><em>SHLD3</em><sup><em>−/−</em></sup>, and <em>BRCA1</em><sup><em>Δ11</em></sup>/RIF1-depleted cells were largely dependent on EXO1 (<a class="figure_ref" href="#fig4">Figures 4</a>E and <a class="workspace-trigger" href="#mmc1">S4</a>I). Thus, distinct domains in 53BP1 recruit effectors to suppress resection by different nucleases.</p></body></html><h3>Shieldin Interferes with RAD51 Loading in BRCA1Δ1153BP1S25A Cells</h3>
<html><body><p id="p0085">To explain the apparent inability of <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells to load RAD51 despite the abundant generation of ssDNA, we speculated that the RIF1/shieldin complex may directly interfere with RAD51 filament formation in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells. Similar to the <em>53BP1</em><sup><em>S25A</em></sup> single mutant (<a class="figure_ref" href="#fig1">Figure 1</a>A), <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> MEFs showed significantly elevated levels of RIF1 foci after IR (<a class="workspace-trigger" href="#mmc1">Figure S5</a>A). To test the role of shieldin in limiting RAD51 foci formation on resected DNA ends, we depleted Shld3 in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> MEFs both by small interfering RNA (siRNA)- and CRISPR/Cas9-mediated deletion. Notably, IR-induced RAD51 foci were restored when Shld3 was depleted from <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> MEFs by either method (<a class="figure_ref" href="#fig5">Figures 5</a>A and <a class="workspace-trigger" href="#mmc1">S5</a>B). Moreover, Shld3 depletion suppressed PARPi hypersensitivity in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> MEFs (<a class="figure_ref" href="#fig5">Figures 5</a>B and <a class="workspace-trigger" href="#mmc1">S5</a>C). We conclude that shieldin blocks RAD51 filament formation post-resection in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells.</p><figure id="fig5"></figure><p id="p0090">SHLD2 features oligonucleotide/oligosaccharide binding (OB) fold domains that bind to ssDNA (<a class="workspace-trigger" href="#bib18">Dev et al., 2018</a>, <a class="workspace-trigger" href="#bib22">Findlay et al., 2018</a>, <a class="workspace-trigger" href="#bib23">Gao et al., 2018</a>, <a class="workspace-trigger" href="#bib24">Ghezraoui et al., 2018</a>, <a class="workspace-trigger" href="#bib26">Gupta et al., 2018</a>, <a class="workspace-trigger" href="#bib35">Mirman et al., 2018</a>, <a class="workspace-trigger" href="#bib38">Noordermeer et al., 2018</a>). Forced targeting of SHLD2 to DSBs (by fusing SHLD2 to the RNF8 FHA domain) was found to suppress RAD51 foci formation in <em>BRCA1</em>/<em>53BP1</em>-deficient cells in a manner dependent on the SHLD2 OB fold (<a class="workspace-trigger" href="#bib38">Noordermeer et al., 2018</a>). Similarly, we found that the expression of FHA-SHLD2 was sufficient to reduce IR-induced RAD51 foci in WT cells only when the OB fold of SHLD2 was intact (<a class="figure_ref" href="#fig5">Figure 5</a>C). However, FHA-SHLD2 did not block IR-induced RPA foci formation (<a class="figure_ref" href="#fig5">Figure 5</a>C). Thus, in the presence of RPA-bound ssDNA, the loading of RAD51 is inhibited when SHLD2 is targeted to chromatin.</p><p id="p0095">Since RNF168-dependent PALB2 recruitment is essential for loading RAD51 in <em>BRCA1/53BP1</em>-deficient cells (<a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>), we examined PALB2 foci formation in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells by the stable overexpression of GFP-PALB2. While IR-induced PALB2 foci were readily detectable in WT cells, PALB2 foci formation was severely compromised in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells (<a class="figure_ref" href="#fig5">Figures 5</a>D and <a class="workspace-trigger" href="#mmc1">S5</a>D). Moreover, PALB2 overexpression did not overcome the defect in RAD51 foci formation (<a class="figure_ref" href="#fig5">Figure 5</a>E). Based on these findings, we speculated that RIF1/shieldin interferes with PALB2/BRCA2 loading on ssDNA, which prevents efficient RAD51 filament formation in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells.</p><p id="p0100">To test this, we targeted PALB2 to chromatin by fusing PALB2 to the FHA domain of RNF8 (FHA-PALB2) (<a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>). Previously, we showed that forced PALB2 targeting to resected DSBs overcomes the severe HR defect in <em>BRCA1</em><sup><em>Δ11/+</em></sup><em>RNF168</em><sup><em>−/−</em></sup> cells (<a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>). While FHA-PALB2 failed to rescue IR-induced RAD51 foci in resection-deficient <em>BRCA1</em><sup><em>Δ11</em></sup> cells, it restored RAD51 foci formation to WT levels in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells that are proficient in end resection (<a class="figure_ref" href="#fig5">Figures 5</a>E and <a class="workspace-trigger" href="#mmc1">S5</a>E). Similarly, whereas FHA-PALB2 failed to mitigate PARPi sensitivity in <em>BRCA1</em><sup><em>Δ11</em></sup> cells or enhance PARPi resistance in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>−/−</em></sup> cells, it markedly reduced the sensitivity of <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells to PARPi (<a class="figure_ref" href="#fig5">Figures 5</a>F and 5G). PALB2 chromatin targeting also significantly reduced RIF1 foci formation (<a class="figure_ref" href="#fig5">Figure 5</a>H), which is consistent with the idea that elevated RIF1/shieldin activity blocks PALB2 function in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells. Finally, the ability of FHA-PALB2 to rescue PARPi resistance and RAD51 foci formation in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> was dependent on its BRCA2-interaction domain (WD40) but not its chromatin association motif (ChAM) (<a class="workspace-trigger" href="#mmc1">Figures S5</a>F and S5G), which is consistent with the idea that PALB2 recruits and is reliant on BRCA2 to assemble RAD51 (<a class="workspace-trigger" href="#bib31">Kowalczykowski, 2015</a>).</p><p id="p0105">To directly examine RAD51 loading on ssDNA, we performed chromatin immunoprecipitation of ssDNA-bound RAD51 (<a class="workspace-trigger" href="#bib6">Brick et al., 2018</a>) at AsiSI-cleaved sites. Consistent with immunofluorescence analyses, forced targeting of PALB2 to chromatin permitted RAD51 loading onto 3′ ssDNA in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells, while it did not nucleate RAD51 filaments at DSB sites in <em>BRCA1</em><sup><em>Δ11</em></sup> cells (<a class="figure_ref" href="#fig5">Figures 5</a>I and <a class="workspace-trigger" href="#mmc1">S5</a>H), presumably because of the lack of sufficient ssDNA. Thus, the post-resection HR defect in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells is caused by an inability to recruit PALB2/BRCA2 to damaged chromatin.</p></body></html><h3>Shieldin Interferes with the RNF168 Ubiquitylation Pathway that Loads RAD51</h3>
<html><body><p id="p0110">We have recently shown that in addition to its requirement for initiating 53BP1 recruitment to DSBs early after DNA damage, the E3 ubiquitin ligase RNF168 acts redundantly with BRCA1 to load PALB2/RAD51 on resected DNA ends (<a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>). We found that the accumulation of RNF168 at DSB sites during S phase is dependent on end resection, as it is abrogated in BRCA1-deficient cells and in WT cells deficient in EXO1/DNA2 (<a class="figure_ref" href="#fig6">Figures 6</a>A and 6B). Moreover, RNF168 loading in S phase is dependent on ATR signaling, as it is severely compromised when cells are pretreated with ATR inhibitor (ATRi) (<a class="figure_ref" href="#fig6">Figure 6</a>C). Because 53BP1 associates with shieldin on H2A-K15-Ub (<a class="workspace-trigger" href="#bib45">Setiaputra and Durocher, 2019</a>), which is also recognized by RNF168 (<a class="workspace-trigger" href="#bib19">Doil et al., 2009</a>, <a class="workspace-trigger" href="#bib47">Stewart et al., 2009</a>), we hypothesized that the block in PALB2 loading in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>25A</em></sup> cells may be due to the defective association of RNF168 with its own mark post-resection. The accumulation of RNF168 foci observed at 4 h after IR was severely compromised in S phase <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells, whereas the initial RNF168 recruitment, necessary for 53BP1/RIF1/shieldin deposition was not altered (<a class="figure_ref" href="#fig6">Figures 6</a>D and 6E). RAD18, which binds H2A-K15-Ub produced by RNF8/RNF168 (<a class="workspace-trigger" href="#bib27">Hu et al., 2017</a>, <a class="workspace-trigger" href="#bib28">Huang et al., 2009</a>, <a class="workspace-trigger" href="#bib40">Panier et al., 2012</a>), also showed defective IRIF in S phase <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells 4 h post-IR (<a class="figure_ref" href="#fig6">Figures 6</a>F and 6H). The deletion of Shld3 in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> rescued both RNF168 and RAD18 recruitment (<a class="figure_ref" href="#fig6">Figures 6</a>G and 6H). Thus, shieldin blocks the RNF168-mediated chromatin ubiquitylation pathway in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells post-resection, thereby compromising BRCA1-independent HR.</p><figure id="fig6"></figure></body></html><h3>RING-less BRCA1 Supports End Resection but Not RAD51 Loading</h3>
<html><body><p id="p0115">The RING domain of BRCA1 possesses ubiquitin ligase activity and is required for stable interaction with BARD1 (<a class="workspace-trigger" href="#bib4">Baer and Ludwig, 2002</a>). Deletion of BRCA1 exon 2 (<em>BRCA1</em><sup><em>Δ2</em></sup>) results in an alternative translation initiation producing an N-terminally truncated BRCA1 protein lacking the RING domain (RING-less BRCA1) (<a class="workspace-trigger" href="#bib20">Drost et al., 2016</a>, <a class="workspace-trigger" href="#bib33">Li et al., 2016</a>, <a class="workspace-trigger" href="#bib52">Wang et al., 2016</a>). RING-less BRCA1 fails to stabilize BARD1 and is defective for HR (<a class="workspace-trigger" href="#bib20">Drost et al., 2016</a>, <a class="workspace-trigger" href="#bib33">Li et al., 2016</a>, <a class="workspace-trigger" href="#bib52">Wang et al., 2016</a>, <a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>). In contrast to <em>BRCA1</em><sup><em>Δ11</em></sup>, we found that <em>BRCA1</em><sup><em>Δ2</em></sup> MEFs exhibited robust levels of DSB resection measured by RPA foci, but failed to form RAD51 foci (<a class="figure_ref" href="#fig7">Figure 7</a>). Since the loss of 53BP1 or RNF168 rescues the RAD51 loading defect in <em>BRCA1</em><sup><em>Δ2</em></sup> cells (<a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>), this indicates that 53BP1 primarily blocks RAD51 loading post-resection in cells expressing mutant BRCA1 lacking the RING domain.</p><figure id="fig7"></figure></body></html><h2>Discussion</h2>
<html><body><p id="p0120">Our study supports a model in which RIF1/shieldin and PTIP associate independently with 53BP1 to regulate distinct end-resection pathways. Although they interact with different phosphorylation sites on 53BP1, RIF1 and PTIP appear to partially occlude each other. This results in the observed increase in IRIF for RIF1 in <em>53BP1</em><sup><em>S25A</em></sup> mutant cells that are incapable of binding PTIP. Nevertheless, elevated RIF1/shieldin activity at DSBs is insufficient to inhibit end resection on its own. Long-range resection is normally mediated by a combination of EXO1 and DNA2, but it is catalyzed predominantly by DNA2 in PTIP-defective <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells (<a class="figure_ref" href="#fig4">Figures 4</a>D and <a class="workspace-trigger" href="#mmc1">S6</a>) and by EXO1 in shieldin-defective <em>BRCA1</em><sup><em>Δ11</em></sup><em>SHLD3</em><sup><em>−/−</em></sup> or <em>BRCA1</em><sup><em>Δ11</em></sup>/RIF1-depleted cells (<a class="figure_ref" href="#fig4">Figures 4</a>E and <a class="workspace-trigger" href="#mmc1">S6</a>). Thus, mutually exclusive phosphorylation-mediated interactions with PTIP and RIF1/shieldin enable 53BP1 to independently suppress end resection by distinct nucleases.</p><p id="p0125">Although the loss of either 53BP1 effector restores end resection to normal levels in BRCA1-deficient cells, they differentially affect the downstream steps of HR. In <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells, the continued presence of shieldin on chromatin post-resection greatly reduces the loading of RAD51 on ssDNA (<a class="workspace-trigger" href="#mmc1">Figure S6</a>). Defective RAD51 loading may in turn promote further hyperresection in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells. Inefficient RAD51 filament assembly is apparently sufficient to rescue the prenatal lethality associated with <em>BRCA1</em><sup><em>Δ11</em></sup> mutation, but it is inadequate to support the normal lifespan in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> mice, even though the cancer rate is not markedly increased. These phenotypes are reminiscent of dominant-negative RAD51 mutations found in Fanconi anemia, which do not produce the typical features of bone marrow failure or cancer predisposition, but instead cause neuronal defects, microcephaly, and sensitivity to DNA-damaging agents (<a class="workspace-trigger" href="#bib2">Ameziane et al., 2015</a>, <a class="workspace-trigger" href="#bib49">Takenaka et al., 2019</a>, <a class="workspace-trigger" href="#bib51">Wang et al., 2015</a>).</p><p id="p0130">How does 53BP1/shieldin block RAD51 filament assembly? RAD51 recombinase activity is facilitated by numerous proteins, including BRCA2, RAD54, and the family of RAD51 paralogs, which act at different stages of HR (<a class="workspace-trigger" href="#bib31">Kowalczykowski, 2015</a>). BRCA2 must first displace RPA from resected DNA ends to enable the subsequent deposition of RAD51 on ssDNA (<a class="workspace-trigger" href="#bib31">Kowalczykowski, 2015</a>). Shieldin, RPA, and BRCA2/PALB2 are OB fold-containing complexes with ssDNA-binding activity (<a class="workspace-trigger" href="#bib45">Setiaputra and Durocher, 2019</a>). An interesting possibility is that RPA and shieldin may form mixed polymers on ssDNA. Although BRCA2 has a higher affinity for ssDNA than RPA, it is possible that BRCA2 cannot efficiently displace shieldin to allow the nucleation of the RAD51 filament. We show that forced targeting of SHLD2 to DSBs is inhibitory to the replacement of RPA by RAD51 (<a class="figure_ref" href="#fig5">Figure 5</a>C). Nevertheless, the block in HR in <em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> cells, associated with the elevation of RIF1/shieldin activity, can be overcome by forced PALB2 loading to chromatin (<a class="figure_ref" href="#fig5">Figures 5</a>E–5H) in a manner dependent on the interaction of PALB2 with BRCA2 (<a class="workspace-trigger" href="#mmc1">Figures S5</a>F and S5G). In <em>BRCA1</em>-deficient cells, the recruitment of PALB2/BRCA2 to ssDNA is dependent on RNF168-driven H2A-K15 ubiquitylation (<a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>). Our data thus suggest that shieldin binding to resected ends can directly compete with RNF168, leading to defective PALB2 accrual, which prevents the nucleation of RAD51 filaments (<a class="workspace-trigger" href="#mmc1">Figure S6</a>). However, a more direct role of shieldin in antagonizing BRCA2 activity cannot be discounted.</p><p id="p0135">The post-resection anti-recombination functions of 53BP1 described herein may be relevant to the mechanism of chemotherapy resistance in a subset of <em>BRCA1</em>-mutated tumors. Germline pathogenic mutations in the RING domain, exemplified by the 185delAG founder mutation commonly found in the Ashkenazi Jewish population, are associated with breast and ovarian cancers and predict poor responses to chemotherapy (<a class="workspace-trigger" href="#bib20">Drost et al., 2016</a>, <a class="workspace-trigger" href="#bib52">Wang et al., 2016</a>). Therapy resistance in this setting has been linked to the overexpression of the hypomorphic RING-less BRCA1 protein, and is associated with partially recovered RAD51 loading and HR (<a class="workspace-trigger" href="#bib20">Drost et al., 2016</a>, <a class="workspace-trigger" href="#bib52">Wang et al., 2016</a>). By contrast, we found that non-cancerous cells expressing RING-less BRCA1 at physiological levels, despite robust end resection, are highly defective in RAD51 loading (<a class="figure_ref" href="#fig7">Figures 7</a>), which can be corrected by the loss of 53BP1 (<a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>). We therefore propose that when the RING-less BRCA1 is overexpressed in cancers, its residual hypomorphic activity is able to partially overcome 53BP1/shieldin binding to resected DNA ends, leading to the partial restoration of HR and the development of chemotherapy resistance.</p><p id="p0140">In summary, the expression of 53BP1<sup>S25A</sup> on a <em>BRCA1</em> mutant background produces a striking uncoupling of end resection from downstream RAD51 loading and suggests that embryonic viability requires end resection but not necessarily efficient HR. These data support a model in which 53BP1 antagonizes two key steps in HR that are normally mediated by BRCA1.</p></body></html><h2>STAR★Methods</h2>
<h3>Key Resources Table</h3>
<html><body><p id="p0145"><div class="tables frame-topbot colsep-0 rowsep-0" id="undtbl1"><div class="groups"><table><thead><tr class="rowsep-1 valign-bottom"><th scope="col">REAGENT or RESOURCE</th><th scope="col">SOURCE</th><th scope="col">IDENTIFIER</th></tr></thead><tbody><tr class="rowsep-1 valign-top"><th colspan="3" scope="row"><strong>Antibodies</strong></th></tr><tr class="valign-top"><td>Rabbit polyclonal anti-53BP1</td><td>Novus Biologicals</td><td>Cat# NB100-305; RRID: AB_10001695</td></tr><tr class="valign-top"><td>Rabbit polyclonal anti-53BP1 (phosho S25)</td><td>Abcam</td><td>Cat# ab70323; RRID:AB_1267592</td></tr><tr class="valign-top"><td>Mouse monoclonal anti-α-Tubulin</td><td>Sigma-Aldrich</td><td>Cat# T-5168; RRID: AB_477579</td></tr><tr class="valign-top"><td>Rabbit polyclonal anti-Phospho-p53 (Ser15)</td><td>Cell Signaling</td><td>Cat# 9284; RRID:AB_331464</td></tr><tr class="valign-top"><td>Mouse monoclonal anti-Chk2</td><td>Millipore</td><td>Cat# 05-649; RRID:AB_2244941</td></tr><tr class="valign-top"><td>Mouse monoclonal anti-GFP</td><td>Roche Applied Science</td><td>Cat# 11814460001; RRID: AB_390913</td></tr><tr class="valign-top"><td>Rabbit polyclonal anti-Rad51</td><td>Abcam</td><td>Cat# ab176458; RRID:AB_2665405</td></tr><tr class="valign-top"><td>Rabbit polyclonal anti-RPA32/RPA2</td><td>Abcam</td><td>Cat# ab10359; RRID:AB_297095</td></tr><tr class="valign-top"><td>Rabbit polyclonal anti-RPA32/RPA2</td><td>Abcam</td><td>Cat# ab76420; RRID:AB_1524336</td></tr><tr class="valign-top"><td>Sheep polyclonal anti-RNF168</td><td>R&amp;D System</td><td>Cat# AF7217; RRID:AB_10971653</td></tr><tr class="valign-top"><td>Mouse monoclonal anti-RAD18</td><td>Millipore</td><td>Cat# ABE1377</td></tr><tr class="valign-top"><td>Mouse monoclonal anti-FLAG (M2)</td><td>Sigma-Aldrich</td><td>Cat# F1804; RRID: AB_262044</td></tr><tr class="valign-top"><td>Rabbit polyclonal anti-RIF1</td><td>Gift from Davide Robbiani</td><td>N/A</td></tr><tr class="valign-top"><td>Mouse monoclonal anti-H2AX (pS139)</td><td>N/A</td><td>Cat# 05-636; RRID:AB_309864</td></tr><tr class="valign-top"><td>Guinea pig polyclonal antibody anti-Tbr1</td><td>Synaptic Systems</td><td>Cat# 328 005; RRID:AB_2620072</td></tr><tr class="valign-top"><td>Rabbit polyclonal antibody anti-Kap1 (Ser 824)</td><td>Bethyl</td><td>Cat# A300-767A; RRID:AB_669740</td></tr><tr class="valign-top"><td>IRDye 680RD Goat anti-Mouse IgG (H+L)</td><td>LI-COR Biosciences</td><td>Cat# 926-68070; RRID:AB_10956588</td></tr><tr class="valign-top"><td>IRDye 800CW Goat anti-Rabbit IgG (H+L)</td><td>LI-COR Biosciences</td><td>Cat# 925-32211; RRID:AB_2651127</td></tr><tr class="valign-top"><td>FITC-Goat anti-Guinea Pig IgG(H+L)</td><td>Jackson ImmunoResearch</td><td>Cat# 106-096-003; RRID:AB_2337418</td></tr><tr class="valign-top"><td>Cy3-Goat anti-Rabbit IgG(H+L)</td><td>Jackson ImmunoResearch</td><td>Cat# 111-166-003; RRID:AB_2338007</td></tr><tr class="valign-top"><td>Alexa Fluor 488 Goat anti-Mouse IgG (H+L)</td><td>Thermo Fisher Scientific</td><td>Cat# A-11001; RRID:AB_2534069</td></tr><tr class="valign-top"><td>Alexa Fluor 568 Goat anti-Mouse IgG (H+L)</td><td>Thermo Fisher Scientific</td><td>Cat# A-11031; RRID:AB_144696</td></tr><tr class="valign-top"><td>Alexa Fluor 568 Goat anti-Rat IgG (H+L)</td><td>Thermo Fisher Scientific</td><td>Cat# A-11077; RRID:AB_2534121</td></tr><tr class="valign-top"><td>Alexa Fluor 488 Chicken anti-Rabbit IgG (H+L)</td><td>Thermo Fisher Scientific</td><td>Cat# A21441; RRID:AB_10563745</td></tr><tr class="valign-top"><td>Alexa Fluor 568 Goat anti-Rabbit IgG (H+L)</td><td>Thermo Fisher Scientific</td><td>Cat# A-11011; RRID:AB_143157</td></tr><tr class="valign-top"><td>Alexa Fluor 568 Donkey anti-Sheep IgG (H+L)</td><td>Thermo Fisher Scientific</td><td>Cat# A21099; RRID:AB_10055702</td></tr><tr class="rowsep-1 valign-top"><td>Purified Rat anti-Mouse CD180 (RP/14)</td><td>BD Biosciences</td><td>Cat# 552128; RRID:AB_394343</td></tr><tr class="rowsep-1 valign-top"><th colspan="3" scope="row"><strong>Bacterial and Virus Strains</strong></th></tr><tr class="valign-top"><td>Bacteria: TOP10 Chemically Competent <em>E. coli</em></td><td>Thermo Fisher Scientific</td><td>Cat# C404006</td></tr><tr class="valign-top"><td>Retrovirus: pCL-ECO</td><td>Addgene</td><td>Cat# 12371; RRID:Addgene_12371</td></tr><tr class="valign-top"><td>Lentivirus: pSpCas9(BB)-2A-Puro (PX459)</td><td>Addgene</td><td>Cat# 48139; RRID:Addgene_48139</td></tr><tr class="valign-top"><td>Mammalian expression pCMV-VsVg envelope lentiviral protein</td><td>Gift from Katrin F. Chua</td><td>N/A</td></tr><tr class="valign-top"><td>Mammalian expression lentiviral packaging pCMV-dR8.2</td><td>Gift from Katrin F. Chua</td><td>N/A</td></tr><tr class="valign-top"><td>Retrovirus: pMX-empty-IRES-GFP-Puro</td><td>Jiri Lukas (<a class="workspace-trigger" href="#bib56">Zong et al., 2015</a>)</td><td>N/A</td></tr><tr class="valign-top"><td>Retrovirus: pMX-empty(no IRES-GFP)-Puro</td><td>Gift from Davide Robbiani</td><td>N/A</td></tr><tr class="valign-top"><td>Retrovirus: pMX-GFP-PALB2-Puro</td><td>(<a class="workspace-trigger" href="#bib56">Zong et al., 2015</a>)</td><td>N/A</td></tr><tr class="valign-top"><td>Retrovirus: pMX-GFP-FHA(RNF8)-PALB2-Puro</td><td>(<a class="workspace-trigger" href="#bib56">Zong et al., 2015</a>)</td><td>N/A</td></tr><tr class="valign-top"><td>Mammalian expression: pCBASceI</td><td>Addgene</td><td>Cat# 26477; RRID:Addgene_26477</td></tr><tr class="valign-top"><td>Mammalian expression: pEGFP-N1</td><td>Jeremy M. Stark</td><td>N/A</td></tr><tr class="valign-top"><td>Adenovirus: Ad5-CMV-eGFP</td><td>Addgene</td><td>N/A</td></tr><tr class="rowsep-1 valign-top"><td>Adenovirus: Ad5-CMV-Cre-eGFP</td><td>Addgene</td><td>N/A</td></tr><tr class="rowsep-1 valign-top"><th colspan="3" scope="row"><strong>Chemicals, Peptides, and Recombinant Proteins</strong></th></tr><tr class="valign-top"><td>Olaparib</td><td>Selleckchem</td><td>Cat# AZD2281</td></tr><tr class="valign-top"><td>ATR inhibitor (AZ20)</td><td>Selleckchem</td><td>Cat# S7050</td></tr><tr class="valign-top"><td>DNA2 inhibitor</td><td>NIH Developmental Therapeutics Program</td><td>Cat# NSC-105808</td></tr><tr class="valign-top"><td>Bloom inhibitor (ML216)</td><td>Sigma-Aldrich</td><td>Cat# SML0661</td></tr><tr class="valign-top"><td>4-Hydroxytamoxifen</td><td>Sigma-Aldrich</td><td>Cat# H7904</td></tr><tr class="valign-top"><td>Doxycycline hyclate</td><td>Sigma-Aldrich</td><td>Cat# D9891</td></tr><tr class="valign-top"><td>Lipopolysaccharide (LPS) from <em>E. coli</em> O111:B4</td><td>Sigma-Aldrich</td><td>Cat# L2630</td></tr><tr class="valign-top"><td>Interleukin 4 (IL-4) from mouse</td><td>Sigma-Aldrich</td><td>Cat# I1020</td></tr><tr class="valign-top"><td>CD43 microbeads (Ly-48)</td><td>Miltenyi Biotec</td><td>Cat# 130-049-801</td></tr><tr class="valign-top"><td>Protein G Magnetic Beads</td><td>Active Motif</td><td>Cat# 104502</td></tr><tr class="valign-top"><td>X-tremeGENE™ 9 DNA Transfection Reagent</td><td>Roche Diagnostics</td><td>Cat# 6365809001</td></tr><tr class="valign-top"><td>Lipofectamine 3000</td><td>Thermo Fisher Scientific</td><td>Cat# L3000015</td></tr><tr class="valign-top"><td>Hoechst 33342</td><td>Thermo Fisher Scientific</td><td>Cat# 62249</td></tr><tr class="valign-top"><td>GeneArt Seamless Cloning Enzyme Mix</td><td>Thermo Fisher Scientific</td><td>N/A</td></tr><tr class="valign-top"><td>Zero Blunt™ PCR Cloning Kit</td><td>Thermo Fisher Scientific</td><td>Cat# K270020</td></tr><tr class="valign-top"><td>PNA probe for telomeres Cy3-(CCCTAA)<sub>3</sub></td><td>PNA Bio</td><td>Cat# F1002</td></tr><tr class="valign-top"><td>DAPI</td><td>Thermo Fisher Scientific</td><td>Cat# 62248</td></tr><tr class="valign-top"><td>EdU</td><td>Thermo Fisher Scientific</td><td>Cat# A10044</td></tr><tr class="valign-top"><td>Colcemid</td><td>Roche Diagnostics</td><td>Cat# 10295892001</td></tr><tr class="valign-top"><td>cOmplete, Mini Protease inhibitor cocktail</td><td>Roche Diagnostics</td><td>Cat# 11836153001</td></tr><tr class="valign-top"><td>Crystal Violet</td><td>Sigma-Aldrich</td><td>Cat# 0775</td></tr><tr class="valign-top"><td>Anti-FLAG M2 magnetic beads</td><td>Sigma</td><td>Cat# M8823; RRID:AB_2637089</td></tr><tr class="valign-top"><td>Trizol</td><td>Invitrogen</td><td>Cat# 15596026</td></tr><tr class="valign-top"><td>Puregene Proteinase K enzyme</td><td>QIAGEN</td><td>Cat# 158920</td></tr><tr class="valign-top"><td>Puregene RNase A Solution</td><td>QIAGEN</td><td>Cat# 158924</td></tr><tr class="valign-top"><td>T4 DNA Polymerase</td><td>NEB</td><td>Cat# M0203L</td></tr><tr class="valign-top"><td>T4 Polynucleotide Kinase</td><td>NEB</td><td>Cat# M0201L</td></tr><tr class="valign-top"><td>DNA Polymerase I, Large (Klenow) Fragment</td><td>NEB</td><td>Cat# M0210L</td></tr><tr class="valign-top"><td>Exonuclease T (ExoT)</td><td>NEB</td><td>Cat# M0265L</td></tr><tr class="valign-top"><td>Exonuclease VII (ExoVII)</td><td>NEB</td><td>Cat# M0379L</td></tr><tr class="valign-top"><td>Klenow Fragment (3′→5′ exo-)</td><td>NEB</td><td>Cat# M0212L</td></tr><tr class="valign-top"><td>Quick Ligation Kit</td><td>NEB</td><td>Cat# M2200L</td></tr><tr class="valign-top"><td>USER enzyme</td><td>NEB</td><td>Cat# M5505L</td></tr><tr class="valign-top"><td>KAPA HiFi HotStart ReadyMix (2X)</td><td>KAPA Biosystems</td><td>Cat# KK2600</td></tr><tr class="valign-top"><td>MyOne Streptavidin C1 Beads</td><td>ThermoFisher</td><td>Cat# 650-01</td></tr><tr class="valign-top"><td>Agencourt AMPure XP beads</td><td>Beckman Coulter</td><td>Cat# A63881</td></tr><tr class="rowsep-1 valign-top"><td>Dynabeads Protein A</td><td>Thermo Fisher Scientific</td><td>Cat# 10002D</td></tr><tr class="rowsep-1 valign-top"><th colspan="3" scope="row"><strong>Critical Commercial Assays</strong></th></tr><tr class="valign-top"><td>Click-IT EdU Alexa Fluor 488 Flow Cytometry Assay Kit</td><td>Thermo Fisher Scientific</td><td>Cat# C10425</td></tr><tr class="valign-top"><td>Click-IT EdU Alexa Fluor 647 Flow Cytometry Assay Kit</td><td>Thermo Fisher Scientific</td><td>Cat# C10634</td></tr><tr class="valign-top"><td>Vector VIP Substrate kit</td><td>Vector laboratories</td><td>Cat# SK-4600; RRID:AB_2336848</td></tr><tr class="valign-top"><td>ApopTag Fluorescein Apoptosis detection kit</td><td>Chemicon</td><td>Cat# S7110</td></tr><tr class="valign-top"><td>CellTiter-Glo® Luminescent Cell Viability Assay</td><td>Promega</td><td>Cat# G7571</td></tr><tr class="valign-top"><td>KAPA Library Quantification Kit</td><td>Kapa Biosciences</td><td>Cat# KK4824</td></tr><tr class="rowsep-1 valign-top"><td>CHEF Mammalian Genomic DNA plug kit</td><td>Bio-Rad</td><td>Cat# 1703591</td></tr><tr class="rowsep-1 valign-top"><th colspan="3" scope="row"><strong>Deposited Data</strong></th></tr><tr class="valign-top"><td>Raw and analyzed data</td><td>This paper</td><td>GSE133808</td></tr><tr class="rowsep-1 valign-top"><td>ChIP-seq for H3K27ac, H3K79me2, H3K27me3</td><td>(<a class="workspace-trigger" href="#bib15">Chronis et al., 2017</a>)</td><td>GSE90893</td></tr><tr class="rowsep-1 valign-top"><th colspan="3" scope="row"><strong>Experimental Models: Cell Lines</strong></th></tr><tr class="valign-top"><td>MEF: Wildtype</td><td>This paper</td><td>N/A</td></tr><tr class="valign-top"><td>MEF: BRCA1<sup>Δ11</sup></td><td>This paper</td><td>N/A</td></tr><tr class="valign-top"><td>MEF: 53BP1<sup>S25A</sup></td><td>This paper</td><td>N/A</td></tr><tr class="valign-top"><td>MEF: 53BP1<sup>−/−</sup></td><td>(<a class="workspace-trigger" href="#bib7">Bunting et al., 2010</a>)</td><td>N/A</td></tr><tr class="valign-top"><td>MEF: BRCA1 <sup>Δ11</sup> 53BP1<sup>−/−</sup></td><td>(<a class="workspace-trigger" href="#bib7">Bunting et al., 2010</a>)</td><td>N/A</td></tr><tr class="valign-top"><td>MEF: BRCA1 <sup>Δ11</sup> 53BP1<sup>S25A</sup></td><td>This paper</td><td>N/A</td></tr><tr class="rowsep-1 valign-top"><td>MEF: BRCA1<sup>Δ2</sup></td><td>(<a class="workspace-trigger" href="#bib7">Bunting et al., 2010</a>)</td><td>N/A</td></tr><tr class="rowsep-1 valign-top"><th colspan="3" scope="row"><strong>Experimental Models: Organisms/Strains</strong></th></tr><tr class="valign-top"><td>Mouse: BRCA1<sup>+/Δ11</sup>. B6/129</td><td>NCI mouse repository</td><td>Strain # 01XC9</td></tr><tr class="valign-top"><td>Mouse: 53BP1<sup>+/</sup><sup>S25A</sup>. B6/129</td><td>This paper</td><td>N/A</td></tr><tr class="valign-top"><td>Mouse: 53BP1<sup>−/−</sup>. B6/129</td><td>(<a class="workspace-trigger" href="#bib53">Ward et al., 2003</a>)</td><td>N/A</td></tr><tr class="valign-top"><td>Mouse: p53<sup>+/−</sup>. B6/129</td><td>Taconic Biosciences</td><td>N/A</td></tr><tr class="rowsep-1 valign-top"><td>Mouse: BRCA1<sup>+/Δ2</sup>. B6/129</td><td>(<a class="workspace-trigger" href="#bib34">Ludwig et al., 1997</a>)</td><td>N/A</td></tr><tr class="rowsep-1 valign-top"><th colspan="3" scope="row"><strong>Oligonucleotides</strong></th></tr><tr class="valign-top"><td>Primers for genotyping 53BP1_S25A allele: Forward, 5′-ggagatggctgagaaagtgc; Reverse, 5′-tcccctggaatggaataaca</td><td>This paper</td><td>N/A</td></tr><tr class="valign-top"><td>sgRNA targeting endogenous mouse Shieldin-3 locus: Sense 5′ CACCGGGAAGTTTGGACTC<br/>ATCGTA; Sense 5′ CACCGGGAAGTTTGG<br/>ACTCATCGTA</td><td>(<a class="workspace-trigger" href="#bib26">Gupta et al., 2018</a>)</td><td>N/A</td></tr><tr class="valign-top"><td>siRNAs targeting endogenous mouse Shieldin-3 locus: Sense 5′ GACUGCACAGUAGAUCUCU<br/>UGGAGU and Antisense 5′ ACUCCAAGAGAU<br/>CUACUGUGCAGUC</td><td>Invitrogen</td><td>N/A</td></tr><tr class="valign-top"><td>shRNAs targeting endogenous mouse Exo1 locus: Pool of antisense 5′-ATAGAACTAGAC<br/>CTACAGAGC, antisense 5′ TTATTCCTCATC<br/>TTAGACGGG and antisense 5′ ATCCGTCAA<br/>ATATGAGAATCG</td><td>Dharmacon</td><td>N/A</td></tr><tr class="rowsep-1 valign-top"><td>Primers to confirm Exo1 knockdown by RT-PCR: Forward: 5′ AGGGGAACAGAACTCC<br/>AAGC, Reverse: 5′ CCAGGAACCTTGTTC<br/>CGTCT</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep-1 valign-top"><th colspan="3" scope="row"><strong>Recombinant DNA</strong></th></tr><tr class="valign-top"><td>Plasmid: FLAG-PALB2</td><td>(<a class="workspace-trigger" href="#bib39">Orthwein et al., 2015</a>)</td><td>N/A</td></tr><tr class="rowsep-1 valign-top"><td>Plasmid: GFP-PALB2</td><td>(<a class="workspace-trigger" href="#bib39">Orthwein et al., 2015</a>)</td><td>N/A</td></tr><tr class="rowsep-1 valign-top"><th colspan="3" scope="row"><strong>Software and Algorithms</strong></th></tr><tr class="valign-top"><td>ZEN 2 (blue edition)</td><td>Zeiss</td><td>https://www.zeiss.com/corporate/int/home.html</td></tr><tr class="valign-top"><td>Gen5 spot analysis</td><td>BioTek</td><td>https://www.biotek.com/</td></tr><tr class="valign-top"><td>Metafer 4</td><td>MetaSystems</td><td>https://metasystems-international.com/</td></tr><tr class="valign-top"><td>Prism 8</td><td>GraphPad</td><td>https://www.graphpad.com/</td></tr><tr class="valign-top"><td>RStudio</td><td>RStudio Team</td><td>https://www.rstudio.com/</td></tr><tr class="valign-top"><td>Bowtie 1.1.2</td><td>(<a class="workspace-trigger" href="#bib32">Langmead et al., 2009</a>)</td><td>https://sourceforge.net/projects/bowtie-bio/files/bowtie/1.1.2/</td></tr><tr class="valign-top"><td>MACS 1.4.3</td><td>(<a class="workspace-trigger" href="#bib54">Zhang et al., 2008</a>)</td><td>https://pypi.org/pypi/MACS/1.4.3</td></tr><tr class="valign-top"><td>UCSC database</td><td>(<a class="workspace-trigger" href="#bib29">Karolchik et al., 2004</a>)</td><td>https://genome.ucsc.edu</td></tr><tr class="valign-top"><td>UCSC genome browser</td><td>(<a class="workspace-trigger" href="#bib30">Kent et al., 2002</a>)</td><td>https://genome.ucsc.edu</td></tr><tr class="valign-top"><td>Bedtools</td><td>(<a class="workspace-trigger" href="#bib41">Quinlan and Hall, 2010</a>)</td><td>https://github.com/arq5x/bedtools2</td></tr><tr class="valign-top"><td>R 3.3.5</td><td><a class="workspace-trigger" href="#bib43">R Development Core Team, 2008</a></td><td>https://www.r-project.org/</td></tr><tr class="valign-top"><td>Sequest</td><td>(<a class="workspace-trigger" href="#bib21">Eng et al., 1994</a>)</td><td>N/A</td></tr><tr class="valign-top"><td>OsiriX v5.7</td><td>Pixmeo</td><td>https://www.osirix-viewer.com/</td></tr><tr class="valign-top"><td>MIM v6.6.5</td><td>MIM Software Inc.</td><td>https://www.mimsoftware.com/</td></tr><tr class="rowsep-1 valign-top"><td>FlowJo (10.1)</td><td>FlowJo LLC</td><td>https://www.flowjo.com/</td></tr><tr class="rowsep-1 valign-top"><th colspan="3" scope="row"><strong>Other</strong></th></tr><tr class="valign-top"><td>BOSC23 retrovirus packaging cells</td><td>ATCC</td><td>Cat# CRL-11270; RRID: CVCL_4401</td></tr><tr class="valign-top"><td>293T lentivirus packaging cells</td><td>ATCC</td><td>Cat# CRL-11268; RRID:CVCL_1926</td></tr><tr class="valign-top"><td>NuPAGE 4-12% Bis-Tris Protein Gels</td><td>ThermoFisher Scientific</td><td>Cat# NP0321BOX</td></tr><tr class="valign-top"><td>Glass Bottom Microwell Dishes</td><td>MatTek Corporation</td><td>Cat# P35G-1.5-14-C</td></tr><tr class="valign-top"><td>FluoroBrite DMEM Media</td><td>Thermo Fisher Scientific</td><td>Cat# A1896701</td></tr><tr class="valign-top"><td>LSM510 confocal microscope</td><td>Zeiss</td><td>N/A</td></tr><tr class="valign-top"><td>Axio Observer Z1 epifluorescence microscope</td><td>Zeiss</td><td>N/A</td></tr><tr class="valign-top"><td>IN Cell Analyzer</td><td>GE Healthcare</td><td>N/A</td></tr><tr class="valign-top"><td>Odyssey® CLx Imaging System</td><td>LI-COR Biosciences</td><td>N/A</td></tr><tr class="valign-top"><td>Lion heart LX automated microscope</td><td>BioTek Instruments</td><td>N/A</td></tr><tr class="valign-top"><td>CyAN ADP cytometer</td><td>Beckman Coulter</td><td>N/A</td></tr><tr class="valign-top"><td>FACSCalibur</td><td>BD Biosciences</td><td>N/A</td></tr><tr class="valign-top"><td>Cytogenetic drying chamber</td><td>Thermotron</td><td>N/A</td></tr><tr class="valign-top"><td>LTQ Orbitrap Velos Pro ion-trap mass spectrometer</td><td>Thermo Fisher Scientific</td><td>N/A</td></tr><tr class="valign-top"><td>Aperio ScanScope XT</td><td>Leica</td><td>N/A</td></tr><tr class="valign-top"><td>Nano Quant Infinite M200 Pro microplate reader</td><td>Tecan</td><td>N/A</td></tr><tr class="valign-top"><td>Mark 1 <sup>137</sup>Cs irradiator</td><td>JL Shepherd</td><td>N/A</td></tr></tbody></table></div></div></p></body></html><h3>Lead Contact and Materials Availability</h3>
<html><body><p id="p0150">Further information and requests for reagents may be directed to and will be fulfilled by the corresponding author Andre Nussenzweig (andre_nussenzweig@nih.gov).</p><p id="p0155">All unique/stable reagents generated in this study are available from the Lead Contact with a completed Materials Transfer Agreement.</p></body></html><h3>Experimental Model and Subject Details</h3>
<html><body><p id="p0160">A targeting construct containing the S25A mutation in murine 53BP1 was constructed by amplification of sequences from BAC clone RP23-179G19 (CHORI, Oakland, CA). Briefly, the left homology arm consisting of 5.1 kB of sequence prior to exon 2 of 53BP1 was inserted into an Entry clone using Gateway recombinational cloning (ThermoFisher, Carlsbad, CA) and flanked by Gateway attB4 and attB1 sites. The right homology arm consisting of exon 2 with an embedded S25A mutation and 3.1 kB of downstream homologous DNA in the subsequent intron was constructed using overlap extension PCR to introduce the appropriate mutation (TCT to GCT) and flanked by Gateway attB2 and attB3 sequences. Homology arm clones were sequenced in their entirety to ensure that no PCR mutations were introduced during amplification. Validated clones were used in a Multisite Gateway recombination reaction along with a selection cassette consisting of an attB1-attB2 flanked reverse orientation PGK promoter-neomycin resistance-BgH polyA fragment and introduced into a Gateway pUC19 destination vector with flanking attR4 and attR3 sites. The final targeting clone consists of the 9.1 kB sequence: left homology arm-(pA-neo-PGK)-exon 2 S25A-right homology arm and is flanked by NotI sites for release of the transgene from the vector backbone. Two derivative targeted ES clones were injected into C57BL/6 blastocysts. The resultant chimeric offspring was backcrossed with wild-type C57BL/6 mice, producing <em>53BP1</em><sup><em>+/S25A</em></sup> animals. Germline transmission of the targeted allele was confirmed by PCR (forward primer: 5′ ggagatggctgagaaagtgc 3′; reverse primer: 5′ tcccctggaatggaataaca 3′). The PGK-neo cassette was removed by crossing to β-actin-Cre transgenic mice (Jackson Laboratory). <em>BRCA1</em><sup><em>+/Δ11</em></sup> mice were obtained from the NCI mouse repository. <em>53BP1</em><sup><em>−/−</em></sup> mice were a gift from Junjie Chen. <em>p53</em><sup><em>−/−</em></sup> mice were obtained from Taconic Biosciences. All mouse breeding and experimentation followed protocols approved by the National Institutes of Health Institutional Animal Care and Use Committee.</p></body></html><h3>Method Details</h3>
<html><body><p id="p0165">Mouse embryonic fibroblasts (MEFs) were generated from E13.5 embryos and grown in Dulbecco’s Modified Eagle’s Medium (DMEM, GIBCO) supplemented with 15% heat-inactivated fetal bovine serum (FBS, Gemini Bio-Products) and 1% penicillin + streptomycin (GIBCO). To establish immortalized MEF cell lines, primary MEFs between passages 2-4 were transiently transfected with a vector encoding SV40 T-antigen (pCMV-SV40T). SV40-immortalized MEFs were routinely cultured in DMEM supplemented with 10 or 15% FBS.</p><p id="p0170">For knockdown of Shieldin-3 in MEFs, CRISPR/Cas9 technology was used by cloning the gene-specific gRNAs into pX459 vector (Sense 5′ CACCGGGAAGTTTGGACTCATCGTA 3′ and Antisense 5′ AAACTACGATGAGTCCAAACTTCCC 3′) as described (<a class="workspace-trigger" href="#bib26">Gupta et al., 2018</a>). Cells were transfected with Lipofectamine following manufacturer’s procedure, selected with puromycin for 48 hours and subjected to single clone isolation. Confirmation of the mutated loci was done through PCR amplification and sequencing of the targeted region.</p><p id="p0175">For transient depletion of endogenous Shieldin-3, siRNA technology (Invitrogen) was used (Sense 5′ GACUGCACAGUAGAUCUCUUGGAGU 3′ and Antisense 5′ ACUCCAAGAGAUCUACUGUGCAGUC 3′). siRNA was transfected with Lipofectamine RNAiMAX (Invitrogen) as per manufacturer’s instructions.</p><p id="p0180">For knockdown of EXO1, a pool of TRC lentiviral short hairpins were used (Dharmacon. 71124 antisense 5′ ATAGAACTAGACCTACAGAGC 3′, 71126 antisense 5′ TTATTCCTCATCTTAGACGGG 3′ and 71127 antisense 5′ ATCCGTCAAATATGAGAATCG 3′). Lentiviral particles were produced by transfecting 293T cells with Lipofectamine-3000 as per manufacturer’s instructions. Forty-eight hours later, viral supernatant was harvested and used to transduce MEFs. Cells were selected with puromycin and used two days later. EXO1 knockdown was confirmed by RT-PCR using the following primers: Forward: 5′ AGGGGAACAGAACTCCAAGC 3′, Reverse2: 5′ CCAGGAACCTTGTTCCGTCT 3′. Samples were run and analyzed on a BioRad CFX96 Real-Time PCR detection system. For microirradiation, MEFs were presensitized in DMEM media containing 0.1 μg/ml of Hoechst 33342 for 60 min before replacing it with phenol red free media containing 5 mM HEPES, and then irradiated with the 364-nm laser line on a LSM510 confocal microscope (Carl Zeiss, Inc.) equipped with a heated stage. Cells were allowed to recover for 1 hour before processing for immunofluorescence.</p><p id="p0185">Retroviral pMX-PIE-based vectors encoding fusion protein of EGFP-FHA-PALB2(del)s were produced as follows: cDNAs corresponding to truncated human PALB2 proteins were produced by PCR amplification from either pDEST-FRT-TO-GFP-PALB2 1-103, -PALB2-deltaCHAM, -PALB2-deltaMRG15, or -PALB2-deltaWD40 (gifts from Dr. Daniel Durocher) then ligated with cDNA corresponding to FHA domain of human RNF8 fused with EGFP (<a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>). The resulting EGFP-FHA-PALB2(del) fragments were subcloned into pMX-PIE vector at the multi-cloning site (MluI/PacI). Infection-competent retroviral particles were assembled in BOSC23 cells co-transfected with the pCL-ECO helper virus. Retroviral supernatant was collected 40–48 h later to transduce MEFs</p><p id="p0190">Western blotting was performed as described previously (<a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>). Briefly, cells were collected and lysed in a buffer containing 50 mM Tris-HCl (pH 7.5), 200 mM NaCl, 5% Tween-20, 0.5% NP-40, 2 mM PMSF, 2.5 mM β-glycerophosphate (all from Sigma-Aldrich) and protease inhibitor cocktail tablet (complete Mini, Roche Diagnostics). Equal amounts of protein were loaded into precast mini-gels (Invitrogen) and resolved by SDS-PAGE. Proteins were blotted onto a nitrocellulose membrane, blocked with 5% membrane blocking agent (GE Healthcare) in TBS and incubated with the corresponding primary antibody. Primary antibodies were used at the following dilutions: anti-53BP1 (1:1000, Novus Biologicals), anti-phospho53BP1-S25 (1:500, Abcam), anti-Tubulin (1:10,000, Sigma- Aldrich), anti-phospho-53(S15) (1:500, Cell Signaling), anti-Chk2 (1:1500, Upstate Biotechnology). Fluorescent secondary antibodies were used at a dilution of 1:15,000 (Li-Cor Biosciences). Detection of protein bands was performed by fluorescence imaging using a Li-Cor Odyssey CLx imaging system (Li-Cor Biosciences).</p><p id="p0195">For immunofluorescence staining, MEFs were grown on 18 mm x 18 mm glass coverslips, and B lymphocytes were attached to slides coated with CellTak (BD Biosciences). Prior to γ-irradiation (<sup>137</sup>Cs Mark 1 irradiator, JL Shepherd), cells were incubated with 10 μM EdU (Invitrogen) for 20 min. Where indicated, cells were additionally pretreated with 1 μM NSC-105808 (DNA2i) or 10 μM AZ20 (ATRi) for 1 hour. Following irradiation, cells were allowed to recover for 1 hour or 4 hours. Pre-extraction and fixation of samples were carried out as previously described (<a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>). The antibodies used for standard immunofluorescence experiments were anti-RIF1 (1:5,000, gift of Davide Robbiani, Rockefeller University), anti-53BP1 (1:1000, Novus), anti-RAD51 (1:250, Abcam), anti-RPA (1:5,000, Abcam), anti-RNF168 (1:100, R&amp;D Systems), anti-RAD18 (1:5000, Millipore), anti-GFP (1:500, Roche). For laser microirradiation experiments, primary antibodies were anti-γH2AX (1:5000, Upstate Biotechnology) and anti-PTIP (1 μg/ml, gift of Kai Ge). Detection was achieved using fluorochrome-conjugated secondary antibodies (Invitrogen). Where indicated, Click-IT chemistry was performed as per manufacturer’s instructions (Thermo Fisher Scientific). Finally, DNA was counterstained with DAPI (Thermo Fisher Scientific). Images were captured at 63 × magnification with an AxioCam MRC5 attached to an Axio Observer Z1 epifluorescence microscope (Zeiss) or at 40 × magnification on a Lionheart LX automated microscope (BioTek Instruments, Inc.). Quantification of nuclear foci and total nuclear intensity was performed using the Gen5 spot analysis software (BioTek). ZEN Blue (Zeiss) was used to quantify fluorescence intensities of laser stripes.</p><p id="p0200">Activated asynchronous B cells and MEFs were treated with 1 μM PARPi (AZD2281, Selleckchem) for 16 hours, subsequently arrested at mitosis with 0.1 μg/ml colcemid (Roche) and metaphase chromosome spreads were prepared as previously described (<a class="workspace-trigger" href="#bib57">Zong et al., 2019</a>). Images were acquired using a Metafer automated scanning and imaging platform (MetaSystems).</p><p id="p0205">To assay for clonogenic survival, MEFs were seeded in 6 cm dishes and treated continuously with 1 μM PARPi (Selleckchem) or exposed to the indicated doses of IR. After 9 days, culture dishes were stained with 0.5% crystal violet. Colonies containing &gt; 50 cells were counted. Clonogenic survival for a given treatment was calculated relative to the plating efficiency in non-treated controls.</p><p id="p0210">To determine cell growth and viability, MEFs were plated in 6-well plates (10,000 per well) and treated continuously with different doses of PARPi for 10 days. The drug-containing medium was replenished every three days and cells were subcultured when they approach confluency. On day 10, cell viability was determined using the CellTiter-Glo Luminescent Cell Viability Assay (Promega) as per manufacturer’s instructions.</p><p id="p0215">To measure SSA, MEFs resuspended in 800 μl of OptiMEM were elecroporated with 15 μg of linearized SA-GFP reporter (<a class="workspace-trigger" href="#bib46">Stark et al., 2004</a>). Cells were selected in puromycin, and puromycin-resistant clones were pooled for analysis. The pools of clones were seeded onto a 24 well dish at 5-7 × 10<sup>5</sup> cells per well, transfected the next day with 0.5 μg of I-SceI expression plasmid (pCBASce) or GFP expression plasmid with 1.8 μl Lipofectamine 2000 (Thermo Fisher Scientific) overnight. GFP<sup>+</sup> cells were analyzed three days after the start of transfection, using a CyAN ADP cytometer (Beckman Coulter). Repair frequencies were normalized to transfection efficiency, using the parallel transfections with the GFP expression vector.</p><p id="p0220">Purification of 53BP1 was performed using an established tandem immunoaffinity method (<a class="workspace-trigger" href="#bib36">Nakatani and Ogryzko, 2003</a>). Flag-HA-53BP1 was stably expressed after transduction of pOZ-Flag-HA-53BP1 retrovirus into HeLa-S cells. Cells were harvested and resuspendend in 5X volume of hypotonic buffer (10 mM Tris, pH 7.3, 10 mM KCl, 1.5 mM MgCl2, 0.2 mM PMSF, 10 mM β-mercaptoethanol, and protease inhibitors (Pierce)). Cells were pelleted by spinning at 2,500 rpm, resuspended in 1X pellet volume of hypotonic buffer and homogenized using a dounce tissue grinder (Wheaton). Nuclear material was pelleted, resuspended in 0.5X pellet volume of low salt buffer (20 mM Tris pH 7.3, 20 mM KCl, 1.5 mM MgCl2, 0.2 mM EDTA, 25% glycerol, 0.2 mM PMSF, 10 mM β-mercaptoethanol, and protease inhibitors), and dounced again. 0.5X pellet volume of high salt buffer (20 mM Tris pH 7.3, 1.2M KCl, 1.5 mM MgCl2, 0.2 mM EDTA, 25% glycerol, 0.2 mM PMSF, 10 mM β-mercaptoethanol, and protease inhibitors) was slowly added to nuclear extract, which was subsequently stirred for 30-45 minutes. Extract was spun down at 14,000 rpm for 30 minutes and the soluble material was dialyzed in BC100 buffer (20 mM Tris pH 7.3, 100 mM KCl, 0.2 mM EDTA, 20% Glycerol, 0.2 mM PMSF, 1 mM β-mercaptoethanol). 53BP1 complex was purified from the nuclear extract using anti-Flag (M2) resin (Sigma), followed by purification using anti-HA (F-7) resin (Santa Cruz) in TAP buffer (50 mM Tris-HCl pH 7.9, 100 mM KCl, 5 mM MgCl2, 10% glycerol, 0.1% NP-40, 1 mM DTT, and protease inhibitors). For both Flag and HA purifications, nuclear extract was rotated with resin for 4 hours, washed extensively with TAP buffer, and eluted with 0.4 mg/mL of Flag or HA peptide (Sigma). After elution, the complex was TCA precipitated and associated proteins were identified by LC-MS/MS using an LTQ Orbitrap Velos Pro ion-trap mass spectrometer (ThermoFisher) and Sequest software.</p><p id="p0225"><em>BRCA1</em><sup><em>Δ11</em></sup><em>53BP1</em><sup><em>S25A</em></sup> and WT littermate control mice were administered PARPi (AZD2281) prepared in 10% DMSO/90% Captisol via oral gavage. Mice were treated with 40 mg/kg of PARPi daily in a volume of approximately 100 μl and monitored daily for changes in health status following PARPi treatment. H&amp;E staining was routinely performed on formalin-fixed, paraffin-embedded tissues. Gut rolls of the small intestine, cecum, and large intestine were digitized with an Aperio ScanScope XT (Leica) at 200 × in a single z-plane. Aperio whole-slide images were evaluated by a board-certified veterinary pathologist.</p><p id="p0230">Sections were photographed at 10 × magnification. Mice were perfused transcardially with 4% paraformaldehyde (PFA), while embryos were drop-fixed in 4% PFA and tissues were cryoprotected in 25% PBS-buffered sucrose solution, embedded in O.C.T. and sectioned sagittally at 10 μm using an HM500M cryostat (Microm). Immunohistochemistry was performed after antigen retrieval. Antibodies used were: anti-Tbr1 (1:250; Synaptic Systems #328005) and anti-phospho KAP1(Ser-824) (1:500; Bethyl Labs, #A300-767A). Immunostaining of active caspase-3 was visualized with a VIP substrate kit (Vector Laboratories) and biotinylated secondary antibody and avidin-biotin complex (Vectastain Elite kit; Vector Laboratories). Sections were counterstained with 0.1% methyl green (Vector Laboratories), dehydrated and mounted with DPX (Fluka). For fluorescence detection, FITC or Cy3 conjugated secondary antibodies (Jackson Immunologicals) were used and nuclei were counterstained with 4’,6-diamidino-2-phenylindole (DAPI) or propidium iodide (Vector Laboratories). TUNEL staining was done using the ApopTag system (Chemicon).</p><p id="p0235">Magnetic resonance imaging (MRI) was performed using a 7 T Bruker ClinScan system (Bruker BioSpin MRI GmbH, Germany) equipped with 12S gradient coil. A two-channel surface coil was used for MR imaging. Animals were anesthetized and maintained with 1.5%–2% isoflurane during MRI sessions. Data analysis was done by manually segmenting the regions and computing volumes using OsiriX (v5.7, Pixmeo, Switzerland).</p><p id="p0240">For the kyphosis study, mice were imaged in the prone position for a 3-bed position X-ray Computed Tomography (CT) (Inveon Multi-Modality PET/CT, Siemens Medical Solutions, Knoxville, TN). X-ray CT step and shoot acquisition parameters were: 80 kVp, 500 μA, 1000 msec per step, 180 steps covering 360-degrees. X-ray CT images were reconstructed using a Feldkamp cone beam algorithm with a Shepp-Logan smoothing filter resulting in 512 × 512 × 1170 matrix (0.08 × 0.08 × 0.08 μm pixel size). Images were linearly calibrated to Hounsfield units (air: −1000 HU, water: 0 HU). The X-ray CT 3D images (orthogonal axial, coronal, and sagittal views) were displayed using a medical image viewer (MIM v 6.6.5, MIM Software Inc, Cleveland, OH) and a pseudo-3D maximum intensity projection (MIP) was implemented to rotate and translate the mouse into perpendicular orientation for analysis</p><p id="p0245">A retrovirus encoding AsiSI (pTRE3G-HA-ER-AsiSI) was stably transduced into the indicated MEF cell lines as previously described (<a class="workspace-trigger" href="#bib10">Canela et al., 2016</a>). Exponentially growing AsiSI-expressing MEFs were treated with 1 μM doxocyclin (DOX) for 24 hours and then, for 5 hours with 4-OHT, resulting in the nuclear translocation of AsiSI. MEFs were harvested and 9 million cells were embedded in agarose plugs, after which END-seq was performed as described (<a class="workspace-trigger" href="#bib11">Canela et al., 2017</a>). END-seq reads were aligned to the mouse (GRCm38p2/mm10) genomes using Bowtie (version 1.1.2) (<a class="workspace-trigger" href="#bib32">Langmead et al., 2009</a>) with parameters -n 3 -k 1 -l 50.</p><p id="p0250">Break intensity was measured by integrating the RPKM values within 100 bp of each AsiSI break site. To quantify the width of maximum resection endpoint (in bp), a sliding window containing twenty 100 bp bins was used, starting from the AsiSI site out to 20 kb to the right side. When more than half of the bins within this sliding window had an RPKM value equal to or lower than the background, then the last bin within the window with a detectable signal over background was regarded as the maximum resection endpoint. Background was determined by the maximum END-seq signal for 100 bp bins more than 20 kb (within 20 kb-30 kb) away from individual AsiSI sites.</p><p id="p0255">To map ssDNA bound by RAD51 and RPA at AsiSI sites (ChIP-SSDS), we first captured DNA bound by RAD51 or RPA. Twenty million cells were harvested for chromatin immunoprecipitation (ChIP) using an anti-RAD51 (Abcam #176458, 10 μg/sample) or anti-RPA antibody (Abcam #10359, 10 μg/sample). ssDNA that spontaneously forms hairpins after heat renaturation was then enriched and sequenced as described (<a class="workspace-trigger" href="#bib6">Brick et al., 2018</a>).</p><p id="p0260">Unless indicated, all data are presented as individual replicates. The total number of replicates, mean and error bars are explained in the figure legends. The statistical tests (Mann-Whitney, Welch’s, Mantel-Cox and Wilcox Rank Sum) and resulting <em>P value</em>s (represented by asterisks) are indicated in the figure legends and/or figure panels and were calculated using GraphPad Prism and R software (ns = p &gt; 0.05; <sup>∗</sup> = p ≤ 0.05; <sup>∗∗</sup> = p ≤ 0.01; <sup>∗∗∗</sup> = p ≤ 0.001; <sup>∗∗∗∗</sup> = p ≤ 0.0001).</p></body></html><h3>Data and Code Availability</h3>
<html><body><p id="p0265">The accession number for the datasets reported in this paper is available at GEO with accession number: GSE133808. https://www-ncbi-nlm-nih-gov.ezp-prod1.hul.harvard.edu/geo/query/acc.cgi?acc=GSE133808.</p></body></html><h2>Acknowledgments</h2>
<html><body><p id="p0270">We thank Anthony Tubbs for comments on the paper; Jennifer Mehalko and Dom Esposito (Protein Expression Laboratory, Frederick National Laboratory for Cancer Research) for transgenic constructs; Karim Baktiar, Diana Haines, and Elijah Edmonson (Pathology/Histotechnology Laboratory, Frederick National Laboratory for Cancer Research) for rodent necropsy, pathology analysis, and imaging; Joseph Kalen and Nimit Patel (Small Animal Imaging Program, Frederick National Laboratory for Cancer Research) for X-ray computed tomography (CT) scan imaging; Jennifer Wise and Kelly Smith for assistance with animal work; Davide Robbiani and Kai Ge for antibodies; Dan Durocher for shieldin constructs; David Goldstein and the CCR Genomics core for sequencing support; and Neil Johnson for discussions. Research in the J.M.S. laboratory is supported by NIH grant R01CA197506. Research in the N.M. laboratory is supported by NIH grant R01 227001. The A.N. laboratory is supported by the Intramural Research Program of the NIH, an Ellison Medical Foundation Senior Scholar in Aging Award (AG-SS- 2633-11), the Department of Defense Idea Expansion (W81XWH-15-2-006) and Breakthrough (W81XWH-16-1-599) Awards, the Alex’s Lemonade Stand Foundation Award, and an NIH Intramural FLEX Award.</p><p id="p0275">E.C., D.Z., N.W., A.S., M.I., R.P., L.C.D., A.K.B., C.M.-D., P.M., A.D., and M.J.K. designed and performed experiments; W.W. and Y.M. analyzed the data; A.C., M.A.B., J.M.S., N.M., P.J.M., and A.N. supervised and provided advice; E.C., D.Z., and A.N. wrote the manuscript with comments from the authors.</p><p id="p0280">The authors declare no competing interests.</p></body></html>